lib/sidekiq-scheduler/sidekiq-adapter.rb

```diff
@@ -0,0 +1,55 @@
+module SidekiqScheduler
+  class SidekiqAdapter
+    def self.fetch_scheduler_config_from_sidekiq(sidekiq_config)
+      return {} if sidekiq_config.nil?
+
+      if SIDEKIQ_GTE_6_5_0
+        sidekiq_config.fetch(:scheduler, {})
+      else
+        sidekiq_config.options.fetch(:scheduler, {})
+      end
+    end
+
+    def self.start_schedule_manager(sidekiq_config:, schedule_manager:)
+      if SIDEKIQ_GTE_6_5_0
+        config[:schedule_manager] = schedule_manager
+        config[:schedule_manager].start
+      else
+        config.options[:schedule_manager] = schedule_manager
+        config.options[:schedule_manager].start
+      end
+    end
+
+    def self.stop_schedule_manager
+      if SIDEKIQ_GTE_6_5_0
+        config[:schedule_manager].stop
+      else
+        config.options[:schedule_manager].stop
+      end
+    end
```

Comment:
are we able to access the config object here somehow without receiving it as an argument of this method?